RetrospectiveExtension.Frontend/dal/processService.tsx

```diff
@@ -77,13 +80,13 @@ export class ProcessService {
     }
 
     public getProcessForCurrentProject = () => {
```

Comment:
if the caller can `await`, this method also can be turned to an `async` method.

when I started coding this extension, I wasn't super familiar with `async-await`, I might missing some obvious awaitable methods.

Reply:
I suggest we keep this change as it's consistent with the rest of the file (all of them uses `.then` style). If we want to switch to async, we can do that in a separate PR in the future.